site/frontend/src/pages/compare/compile/compile-page.vue

```diff
@@ -222,6 +258,14 @@ const filteredSummary = computed(() => computeSummary(comparisons.value));
     :selected-metric="selector.stat"
     :metrics="benchmarkInfo.compile_metrics"
   />
+  <div v-if="comparesIdenticalCommits">
+    Self-compare backend:
```

Comment:
Maybe something like

```suggestion
    Compare codegen backends for this commit:
```

maybe with the given commit in the label, like "for commit $sha"